### PR TITLE
[RND-405] - Adding error message to Malformed body messages

### PR DESCRIPTION
**Pull request by stevenarnold-ed-fi**

### CODE REVIEW

**Comment by bradbanister** [comment]
Looks like there is a conflict with the old/new dorny reporter action version.

File: Meadowlark-js/packages/meadowlark-authz-server/src/handler/RequestToken.ts [hunk 1]
```diff
@@ -68,14 +68,14 @@ function parseRequestTokenBody(authorizationRequest: AuthorizationRequest): Pars
     try {
       unvalidatedBody = querystring.parse(authorizationRequest.body);
     } catch (error) {
-      Logger.debug(`${moduleName}.parseRequestTokenBody: Malformed body`, authorizationRequest.traceId);
+      Logger.debug(`${moduleName}.parseRequestTokenBody: Malformed body - ${error.message}`, authorizationRequest.traceId);
       return { isValid: false, failureMessage: 'Malformed body' };
```
Comment (stephenfuqua) [inline Meadowlark-js/packages/meadowlark-authz-server/src/handler/RequestToken.ts:72]: I was a little surprised that this was not updated. But I might be ok with that for the auth server.

Comment (stephenfuqua) [reply Meadowlark-js/packages/meadowlark-authz-server/src/handler/RequestToken.ts:72]:
```suggestion
      return { isValid: false, failureMessage: `Malformed body: ${error.message}` };
```

File: Meadowlark-js/packages/meadowlark-authz-server/src/handler/RequestToken.ts [hunk 2]
```diff
@@ -68,14 +68,14 @@ function parseRequestTokenBody(authorizationRequest: AuthorizationRequest): Pars
...
     }
   } else {
     try {
       unvalidatedBody = JSON.parse(authorizationRequest.body);
     } catch (error) {
-      Logger.debug(`${moduleName}.parseRequestTokenBody: Malformed body`, authorizationRequest.traceId);
+      Logger.debug(`${moduleName}.parseRequestTokenBody: Malformed body - ${error.message}`, authorizationRequest.traceId);
       return { isValid: false, failureMessage: 'Malformed body' };
```
Comment (stephenfuqua) [inline Meadowlark-js/packages/meadowlark-authz-server/src/handler/RequestToken.ts:79]:
```suggestion
      return { isValid: false, failureMessage: `Malformed body: ${error.message}` };
```

**Review by stephenfuqua — APPROVED** [review]

**Pull request merged**